osx/spec/spec_helper.rb

```diff
@@ -0,0 +1,3 @@
+require 'serverspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.